lua/wikis/sideswipe/MatchSummary.lua

```diff
@@ -33,40 +32,14 @@ end
 ---@param opponent2 standardOpponent
 ---@return Html
 function Header:createScoreDisplay(opponent1, opponent2)
-	local function getScore(opponent)
-		local scoreText
-		local isWinner = opponent.placement == 1 or opponent.advances
-		if opponent.placement2 then
-			-- Bracket Reset, show W/L
-			if opponent.placement2 == 1 then
-				isWinner = true
-				scoreText = 'W'
-			else
-				isWinner = false
-				scoreText = 'L'
-			end
-		elseif opponent.extradata and opponent.extradata.additionalScores then
-			-- Match Series (Sets), show the series score
-			scoreText = (opponent.extradata.set1win and 1 or 0)
-					+ (opponent.extradata.set2win and 1 or 0)
-					+ (opponent.extradata.set3win and 1 or 0)
-		else
```

Comment:
Is this really part of the default implementation?

Reply:
It was incorrectly added to sideswipe https://github.com/Liquipedia/Lua-Modules/pull/5929/files#diff-fb6d39714d611ff9e90166aa36bc9acd7c6f25ffd34014fc91745a9ff599e59a so this just reverts this fucntion on ss